### PR TITLE
Update op_level_cost_estimator.cc

### PR DESCRIPTION
**Pull request by MikalaiDrabovich**
wccftech.com/nvidia-gtx-1080-gp104-die-shot
techpowerup.com/gpu-specs/geforce-rtx-2080-ti.c3305

### CODE REVIEW

File: tensorflow/core/grappler/costs/op_level_cost_estimator.cc [hunk 1]
```diff
@@ -401,9 +401,11 @@ DeviceInfo OpLevelCostEstimator::GetDeviceInfo(
     } else if (architecture < "6") {
       // Maxwell
       cores_per_multiprocessor = 128;
+    } else if (architecture < "7") {
+      // Pascal
+      cores_per_multiprocessor = 128;
     } else {
-      // Pascal (compute capability version 6) and Volta (compute capability
-      // version 7)
+      // Turing
```
[the remark this answers is not in the document]
Comment (jlebar) [reply tensorflow/core/grappler/costs/op_level_cost_estimator.cc]: It's different for SM 6.0 vs SM 6.1 and 6.2:

https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#architecture-6-x

7.x (Turing or Volta) is apparently also 64 cores per SM: https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#architecture-7-x

"Cores per SM" is sort of a meaningless figure, but I think it *should* be fair to use it to compute gflops, if that's all you're doing.

**Review by jlebar — CHANGES_REQUESTED** [review]
I don't think this is correct.

**Comment by MikalaiDrabovich** [comment]
yes, as per @jlebar link, SM 6.1 and 6.2 should have 128, not 64 'cores', otherwise grappler will probably optimize for 2x lower gflops then available.

**Comment by rmlarsen** [comment]
@MikalaiDrabovich will you update the PR to reflect the documentation from NVIDIA?

**Comment by rthadur** [comment]
> @MikalaiDrabovich will you update the PR to reflect the documentation from NVIDIA?

@MikalaiDrabovich gentle ping to update PR

**Comment by rthadur** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**